edsdme/scripts/utils.js

```diff
@@ -150,6 +150,73 @@ export function redirectLoggedinPartner() {
   window.location.assign(target);
 }
 
+export async function getRenewBanner() {
+  const programType = getCurrentProgramType();
+  const accountExpiration = getPartnerDataCookieValue(programType, 'accountexpiration');
```

Comment:
This value is returned from runtime as  `accountAnniversary (accountanniversary)`
```
"partnerData": {
        "CPP": {
            "accountAnniversary": "2025-07-17T00:00:00.000Z",
            "company": "Yugo Stage CPP  Gold United Kingdom",
            "firstName": "Yugo Stage CPP",
            "lastName": "Gold United Kingdom",
            "level": "Gold",
            "permissionRegion": "United Kingdom,Europe West",
            "status": "MEMBER"
        }
    },
```

Reply:
@zagi25 Thank you, will change the name